.werft/build.js

```diff
@@ -102,7 +107,7 @@ async function build(context, version) {
     if (withInstaller || publishRelease) {
         exec(`leeway build --werft=true -c ${cacheLevel} ${dontTest ? '--dont-test':''} -Dversion=${version} -DimageRepoBase=${imageRepo} install:all`, buildEnv);
     }
-    exec(`leeway build --werft=true -Dversion=${version} -DremoveSources=false -DimageRepoBase=${imageRepo}`, buildEnv);
+    exec(`leeway build --werft=true -Dversion=${version} -DgitpodDomain=${domain} -DremoveSources=false -DimageRepoBase=${imageRepo}`, buildEnv);
```

Comment:
Can you make the Domain Name a parameter or env var of the IDE image? 
Introducing it as built tim variance is problematic because the build artefacts should be installable on any domain.

Reply:
I totally agree. I have made some steps in that direction here: https://github.com/gitpod-io/gitpod/tree/clu/openvsx-proxy-2
However, that doesn't work yet that well.